### DISCUSSION TITLE
Embed editor significantly faster than regular editor

### DISCUSSION THREAD

**Discussion by mattrossman** [codesandbox/codesandbox-client#5644] [General] [answered]
Has anyone else noticed that the `/embed/` editor is significantly faster than the regular `/s/` editor? It feels like the difference between using CRA and vite. The regular editor takes around a second to reload, whereas the embed editor updates almost as fast as I can type.

Is there any reason we don't get this performance in the regular editor? I see a related issue from 2 years ago in #274 but it was closed without resolution.

Just compare the difference between typing changes in this:
https://codesandbox.io/s/new

and this:
https://codesandbox.io/embed/new

It's night and day!

**Comment by CodeSandboxTeam** [comment, marked as answer]
Hey Matt,


Yes, embed are meant to be faster than the normal sandbox experience. 
It's because many things (eslint, vscode things, sharing/Github/other features) are not started in the embed one. If you do not need the full experience, you can use embed for purely coding.


Andras



--- original message ---
On April 9, 2021, 2:01 AM GMT+2 ***@***.*** wrote:



Has anyone else noticed that the /embed/ editor is significantly faster than the regular /s/ editor? It feels like the difference between using CRA and vite. The regular editor takes around a second to reload, whereas the embed editor updates almost as fast as I can type.

Is there any reason we don't get this performance in the regular editor? I see a related issue from 2 years ago in #274 but it was closed without resolution.

Just compare the difference between typing changes in this:

https://codesandbox.io/s/new

and this:

https://codesandbox.io/embed/new

It's night and day!

—

You are receiving this because you are subscribed to this thread.

Reply to this email directly, view it on GitHub, or unsubscribe.
--- end of original message ---
